extensions/query-history/README.md

```diff
@@ -0,0 +1,16 @@
+# Query History *(preview)*
+
+Adds a Query History panel for viewing and running past executed queries.
```

Comment:
Could probably add a bit more details and screenshot here since this will be on marketplace...of course that can be done separately prior to release.

Reply:
Yeah I'll follow up with that later - will make an issue for it to track. 

---
In reply to: [310679305](https://github.com/microsoft/azuredatastudio/pull/6579#discussion_r310679305) [](ancestors = 310679305)